### PR TITLE
Simple Random Sampling (SRS)

### PR DESCRIPTION
**Pull request by rofinn**
Extension to #36 

- Fixup Project.toml
- Update to latest API
- Rename HotDeck to SRS
- Add AbstractRNG field for testing
- Updated doctests

@paulstey for review

### CODE REVIEW

**Comment by paulstey** [comment]
This all looks great! Thanks, @rofinn!!

File: src/Impute.jl [hunk 1]
```diff
@@ -1,11 +1,13 @@
 module Impute
 
 using IterTools
+using Random
 using Statistics
 using StatsBase
 using Tables: Tables, materializer, istable
 
 import Base.Iterators: drop
+import DataFrames: dropmissing
```
Comment (rofinn) [inline src/Impute.jl]: Why was this needed again?
